src/taskProvider.ts

```diff
@@ -276,18 +276,21 @@ export class TaskProvider implements vscode.TaskProvider {
   private runDevToolsImageExec(): string {
     let exec = "";
 
+    const conf = vscode.workspace.getConfiguration("ledgerDevTools");
```

Comment:
If you put the scope to `resource` you will have to provide the folderUri where you want to fetch the setting : 

```
  const conf = vscode.workspace.getConfiguration("ledgerDevTools", this.appFolderUri);
```